src/table/column-filter.scss

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../menu/index';
+
+@mixin iui-column-filter {
+  @include iui-dropdown;
+  padding: $iui-baseline $iui-sm;
+  width: fit-content;
+
+  .iui-icon-overlay {
+    position: relative;
+
+    > .iui-input {
+      padding-right: $iui-l * 2;
+    }
+
+    > .iui-borderless {
+      position: absolute;
+      right: 0;
+      border-top-left-radius: 0;
+      border-bottom-left-radius: 0;
+    }
+  }
+
+  .iui-right-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+
+    > .iui-date-time {
+      display: flex;
+
+      > .iui-input-container > .iui-input {
```

Comment:
Do we need this one?
![image](https://user-images.githubusercontent.com/36186912/116521344-873b1600-a8dc-11eb-907e-ed658a13c775.png)
It is very wide and for me this one is enough when we add time selection to date picker:
![image](https://user-images.githubusercontent.com/36186912/116521422-9cb04000-a8dc-11eb-9ee9-34b69d4215ec.png)

Reply:
I can remove it, will certainly simplify the CSS.  We will need to update the standard documentation to remove the wider option.  That option was implemented before we had a date + time combo picker.

Reply:
It's been removed.